cudax/include/cuda/experimental/__container/uninitialized_buffer.cuh

```diff
@@ -186,11 +186,17 @@ public:
   }
 
   //! @brief Returns the size of the buffer
-  _CCCL_NODISCARD _CCCL_HOST_DEVICE constexpr size_type size() const noexcept
```

Comment:
We should rather drop the `_CCCL_HOST_DEVICE`

conventionally the `_CCCL_HOST_DEVICE` macro commes before constexpr as all other attributes